.cursorrules

```diff
@@ -0,0 +1,6 @@
+## Sample JSON
```

Comment:
I mentioned this before: this doesn't look like something that should be checked in. shouldn't this be gitignore'd?